extensions/labeled-response.md

```diff
@@ -67,8 +64,6 @@ If a response consists of more than one message, and the `batch` capability is e
 * An existing type applicable to the entire response
 * `draft/labeled-response`
 
-If no response is required, an empty batch MUST be sent.
-
 If `batch` has not been enabled, the tag MAY be included on only one of the messages in the response.
```

Comment:
May be worth mentioning that using the `ACK` message is explicitly required here, esp with the above line being removed?

> If `batch` has not been enabled, the tag MAY be included on only one of the messages in the response. If `batch` has not been enabled and there are no messages in the response, the `ACK` message MUST be returned.

eh, no worries either way though, feel free to add or leave

Reply:
Maybe that instead should be mentioned in "The `ACK` command" section, that it should be sent regardless of whether or not the server sends batches? I think it's obvious that people should expect it either way, but it wouldn't hurt to clarify.

Reply:
See, this is why I'd rather just always require ACK.